tests/common.go

```diff
@@ -3157,3 +3181,173 @@ func init() {
 	logrus.StandardLogger().Hooks.Add(log.NewHook())
 	logrus.SetOutput(os.Stdout)
 }
+
+//CreateJiraIssueWithLogs creates a jira issue and copy logs to nfs mount
+func CreateJiraIssueWithLogs(issueDescription, issueSummary string) {
```

Comment:
This function should return an error if `jirautils.CreateIssue` fails.

Reply:
done